### DISCUSSION TITLE
About Admin

### DISCUSSION THREAD

**Discussion by chandramouli35** [RishiBakshii/mern-ecommerce-reduxToolkit#16] [Q&A] [answered]
How can we check admin side

**Comment by RishiBakshii** [comment, marked as answer]
hi @chandramouli35 thankyou for raising an issue here
 for exploring the admin side, setup this project on your local machine and after make a new account and update "isAdmin":true in mongo db

**Comment by shargtech** [comment]
Hello How can i contact you , please mai me shargtech@gmail.com
